controllers/certificate_request_controller.go

```diff
@@ -175,39 +180,34 @@ func (r *CertificateRequestReconciler) Reconcile(ctx context.Context, req ctrl.R
 
 		c := issuerStatus.GetCondition(tcsapi.IssuerConditionReady)
 		if c == nil || c.Status == v1.ConditionFalse {
-			return ctrl.Result{Requeue: true, RequeueAfter: time.Minute}, errIssuerNotReady
```

Comment:
Is requeuing a separate modification, not related to the double status update? Please make it a new commit to make easier to track where the changes happen. Why's the one minute timeout?

Reply:
> Is requeuing a separate modification, not related to the double status update? Please make it a new commit to make easier to track where the changes happen.

This change is to avoid unnecessary reconcile re-queue. I will pull that in a separate commit

>  Why's the one minute timeout?

The initial idea was to re-queue the request in case of internal errors. But anyways, we update the object status in case of error, which anyways leads to a new reconcile call.

Reply:
> This change is to avoid unnecessary reconcile re-queue. I will pull that in a separate commit

Done.